ui-admin/src/navbar/StudySidebar.tsx

```diff
@@ -0,0 +1,94 @@
+import { studyPublishingPath, studyUsersPath } from '../study/StudyRouter'
+
+/** shows menu options related to the current study */
+export const StudySidebar = ({ study, portalList, portalShortcode }:
+                                 {study: Study, portalList: Portal[], portalShortcode: string}) => {
+  const navigate = useNavigate()
+  /** updates the selected study -- routes to that study's homepage */
+  const setSelectedStudy = (portalShortcode: string, studyShortcode: string) => {
+    navigate(studyParticipantsPath(portalShortcode, studyShortcode, 'live'))
+  }
+  const navStyleFunc = ({ isActive }: {isActive: boolean}) => {
+    return isActive ? { background: '#567' } : {}
+  }
+
+  return <div className="pt-3">
+    <StudySelector portalList={portalList} selectedShortcode={study.shortcode} setSelectedStudy={setSelectedStudy}/>
+    <div className="text-white">
+      <CollapsableMenu header={'Research Coordination'} content={<ul className="list-unstyled">
+        <li className="mb-3">
+          <NavLink to={studyParticipantsPath(portalShortcode, study.shortcode, 'live')}
+            className="text-white p-1 rounded" style={navStyleFunc}>Participant List</NavLink>
+        </li>
+        <li className="mb-3">
+          <NavLink to={studyKitsPath(portalShortcode, study.shortcode, 'live')}
+            className="text-white p-1 rounded" style={navStyleFunc}>Biologistics</NavLink>
+        </li>
+        <li>
+          <NavLink to={studyEnvMailingListPath(portalShortcode, study.shortcode, 'live')}
+            className="text-white p-1 rounded" style={navStyleFunc}>Mailing list</NavLink>
+        </li>
+      </ul>}/>
+      <CollapsableMenu header={'Analytics & Data'} content={<ul className="list-unstyled">
+        <li className="mb-3">
+          <NavLink to={studyEnvMetricsPath(portalShortcode, study.shortcode, 'live')}
+            className="text-white p-1 rounded" style={navStyleFunc}>Participant Analytics</NavLink>
+        </li>
+        <li className="mb-3">
+          <NavLink to={studyEnvDataBrowserPath(portalShortcode, study.shortcode, 'live')}
+            className="text-white p-1 rounded" style={navStyleFunc}>Data Export</NavLink>
+        </li>
+        <li>
+          <NavLink to={studyEnvDatasetListViewPath(portalShortcode, study.shortcode, 'live')}
+            className="text-white p-1 rounded" style={navStyleFunc}>Terra Data Repo</NavLink>
+        </li>
+      </ul>}/>
+      <CollapsableMenu header={'Design & Build'} content={<ul className="list-unstyled">
+        <li className="mb-3">
+          <NavLink to={studyEnvSiteContentPath(portalShortcode, study.shortcode, 'sandbox')}
+            className="text-white p-1 rounded" style={navStyleFunc}>Website</NavLink>
+        </li>
+        <li className="mb-3">
+          <NavLink to={studyEnvFormsPath(portalShortcode, study.shortcode, 'sandbox')}
+            className="text-white p-1 rounded" style={navStyleFunc}>Forms & Surveys</NavLink>
```

Comment:
Nit, but I've noticed you've been deliberate about using `&amp;` in other places.
```suggestion
            className="text-white p-1 rounded" style={navStyleFunc}>Forms &amp; Surveys</NavLink>
```